guard-lambda/README.md

```diff
@@ -123,12 +123,50 @@ The Lambda version of the tool is a lightweight wrapper around the core [cfn-gua
 
 ### Building and deploying
 
+#### Guided
+
 1. Make sure docker is running
 2. Navigate to `guard-lambda` directory and run `sam build --use-container` to build the code for the Lambda function
 3. Run `sam deploy --guided` and complete the interactive workflow. This workflow will create a CloudFormation changeset and deploy it
 4. Once it succeeds, the name of the function will be shown in the `CloudFormationGuardLambdaFunctionName` output
```

Comment:
If we are asking user to pass a parameter for name this line will change, correct?

Reply:
If a `samconfig.toml` is not being used, you are correct. `sam deploy` will need this added. I will update the PR.

Reply:
I misunderstood - no, I don't think this will need to change. The output does not change by having a stack parameter. Also, if following the guided approach, it should be managed through the interactive process. The parameter is not required, so already existing consumers will not need to add it.